src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheckTest.java

```diff
@@ -169,13 +199,8 @@ public void testConditions() throws Exception {
         checkConfig.addAttribute("tokens", "LITERAL_ELSE, LITERAL_CASE, LITERAL_DEFAULT");
         checkConfig.addAttribute("allowSingleLineStatement", "true");
         final String[] expected = {
-            "29: " + getCheckMessage(MSG_KEY_NEED_BRACES, "case"),
-            "35: " + getCheckMessage(MSG_KEY_NEED_BRACES, "case"),
```

Comment:
why this not violation any more ?
https://github.com/checkstyle/checkstyle/blob/3fbffc4c7736af507513e86e93b136df5301d04a/src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/InputNeedBracesConditional.java#L35

case is very weird but it should be:
```
        case 1:  {System.lineSeparator();}
        case 2: { break;}
```
and we let user think about reason of such weird code.

Reply:
>why this not violation any more ?

This
> checkConfig.addAttribute("allowSingleLineStatement", "true");

Reply:
Ok